tutorial/20_recipes/007_optuna_callback.py

```diff
@@ -0,0 +1,53 @@
+"""
+.. _optuna_callback:
+
+Callback for Study.optimize
+===========================
+
+This tutorial showcases how to use & implement Optuna ``Callback`` for :func:`~optuna.study.Study.optimize`.
+By callback, we mean a callable that takes :class:`~optuna.study.Study` and :class:`~optuna.trial.FrozenTrial` and does some work.
+
+Note that callbacks in this tutorial and ``*PruningCallback``'s of :mod:`optuna.integration` are completely different.
```

Comment:
Here, you said they are different, but how about describing how they are different? e.g. explain definition or usage of `Callback for Study.optimize` and `*PruningCallback` and show their difference.